ispcrt/detail/gpu/GPUDevice.h

```diff
@@ -53,3 +53,11 @@ struct GPUDevice : public base::Device {
 };
 
 } // namespace ispcrt
+
+// Expose API of GPU device solib for dlsym.
+extern "C" {
```

Comment:
Why not all methods which are defined as extern "C" in .cpp file are declared here?

Reply:
This is a device header, two extra methods are in the context header. GPUDevice.cpp contains both context and device functions.

Reply:
I see now, thanks